py/src/main.py

```diff
@@ -15,11 +18,19 @@ def main(argv: List[str]) -> int:
     try:
         gameboy = GameBoy(args)
         gameboy.run()
+    except (InvalidOpcode, InvalidRamRead, InvalidRamWrite) as e:
+        raise GameException(e) 
+    except (Quit, Timeout, UnitTestFailed, UnitTestPassed) as e:
+        raise ControlledExit(e)
+    except (RomMissing, LogoChecksumFailed, HeaderChecksumFailed) as e:
+        raise UserException(e)
```

Comment:
`InvalidOpcode` is already a sub-class of `GameException`, no need to catch the specific type and raise the generic type (and the same for the others - ie, these highlighted lines don't really add anything, so they can go)

Reply:
Hey sure thing, I'll remove those lines and push up again